src/crate/crash/command.py

```diff
@@ -472,6 +494,34 @@ def main():
     cmd.exit()
     sys.exit(cmd.exit_code)
 
+def test_main():
```

Comment:
Would it be possible to extract some smaller methods from `main()` - which are then suitable to be called from a test?